aces/pipeline_scripts/generate_spw33_commands.py

```diff
@@ -36,9 +36,9 @@
                 print(f"Adding {key}")
                 spw33pars = commands[key]['tclean_cube_pars']['spw35']
                 spw33pars['imagename'] = spw33pars['imagename'].replace('35', '33')
-                spw33pars['nchan'] = nchan
-                spw33pars['start'] = start
-                spw33pars['width'] = chwid
+                spw33pars['nchan'] = -1
+                spw33pars['start'] = ""
+                spw33pars['width'] = ""
                 spw33pars['threshold'] = '0.01Jy'
```

Comment:
these are the biggest change - we're switching to defaulting to let tclean decide nchan